tests/components/login/login_controller/login_controller.test.jsx

```diff
@@ -62,4 +64,35 @@ describe('components/login/LoginController', () => {
 
         expect(wrapper).toMatchSnapshot();
     });
+
+    it('should show session expiry notification', () => {
+        const props = {
+            ...baseProps,
+            initializing: false,
+        };
+
+        LocalStorageStore.setWasLoggedIn(true);
+        const wrapper = shallowWithIntl(
+            <LoginController {...props}/>
+        ).dive();
+
+        expect(wrapper).toMatchSnapshot();
```

Comment:
I fought hard to make this an inline snapshot, but two problems:
* The component is too fat to inline the whole thing, and `childAt().childAt()...` makes the test too brittle.
* My editor kept removing crucial spaces in the inline snapshot that caused pointless differences.

Until we refactor the component, I've opted for external snapshots here.